Jenkinsfile

```diff
@@ -141,6 +141,7 @@ node {
         stage("clang-debug-coverage") {
           if (env.BRANCH_NAME == 'master' || full_ci) {
             sh "export CCACHE_BASEDIR=`pwd`; ./scripts/coverage.sh --generate_badge=true --launcher=ccache"
+            sh "find coverage -type d -exec chmod +rx {} \\;"
```

Comment:
Woudn't `chmod -R +rX coverage` do the same?

Reply:
I believe @nilsthamm found this exact fix somewhere, but I don’t see why your suggestion shouldn’t have the same effect. On the other hand, does it make any difference? Both should be understandable.

Reply:
Instead of invoking 1+n processes, you only have one. It's not really a big thing, but since I was unaware of the `X` flag for a long time, I just wanted to point it out.

No reason to change anything.